testing-internal/src/main/java/com/linecorp/centraldogma/testing/internal/TestUtil.java

```diff
@@ -35,11 +40,15 @@
     public static <T> void assertJsonConversion(T value, Class<T> valueType, String json) {
         assertThatJson(json).isEqualTo(value);
         try {
-            assertEquals(value, Jackson.readValue(json, valueType));
+            assertThat(Jackson.readValue(json, valueType)).isEqualTo(value);
         } catch (IOException e) {
             throw new IOError(e);
         }
     }
 
+    public static String getMethodName(TestInfo testInfo) {
```

Comment:
nit: How about renaming this to `normalizedDisplayName` for clarifying the method behavior? 😀